src/acl.c

```diff
@@ -2790,6 +2790,14 @@ void aclCommand(client *c) {
             return;
         }
 
+        if (doesCommandHaveKeys(cmd) &&
```

Comment:
why do we need to check `doesCommandHaveKeys`?
i think it's ok to return arity error before proceeding, even if the command doesn't have keys.